src/components/generics/overview-chart/overview-chart.tsx

```diff
@@ -76,7 +76,10 @@ const OverviewChart: React.FC<Props> = ({ displayedData, isLoading }) => {
         >
           { selectedData.name }
         </Typography>
-        { selectedData.project !== undefined && renderCustomizedTooltipRow(strings.project, selectedData.project as number, theme.palette.success.main) }
+        {/* eslint-disable */}
```

Comment:
Why is eslint-disabled here?

Reply:
To prevent errors from having too long line. At the time of writing this I wasn't aware that you're able to disable individual rules.
Turned out to be redundant after these requests tho.